web/src/features/hfiCalculator/formatTableAsCSV.test.ts

```diff
@@ -0,0 +1,691 @@
+import { FormatTableAsCSV } from 'features/hfiCalculator/FormatTableAsCSV'
+import { DateTime } from 'luxon'
+import { FireCentre } from 'api/hfiCalcAPI'
+import { StationDaily } from 'api/hfiCalculatorAPI'
+describe('RowManager', () => {
+  const fireCentres: Record<string, FireCentre> = {
+    0: {
+      id: 12,
+      name: 'Kamloops Fire Centre',
```

Comment:
Please add a name that contains some csv characters that need to be escaped. For example, 'Kamloops, Fire Centre' or 'Kamloops \' fire " Centre' to make sure escaping is working.